src/abstractdataframe/abstractdataframe.jl

```diff
@@ -587,22 +590,30 @@ function nonunique(df::AbstractDataFrame)
     res
 end
 
+nonunique(df::AbstractDataFrame, cols::Any) = nonunique(df[[cols]])
+nonunique{T <: Any}(df::AbstractDataFrame, cols::AbstractVector{T}) = nonunique(df[cols])
```

Comment:
I think the default method (i.e. `cols::Any`) should be the "vector" one, and for the "scalar" case the parameters should be explicitly listed (`col::Union{Real,Symbol}` as in DataFrame's `ColumnIndex`). Otherwise `Colon()` could be incorrectly dispatched.